src/test/java/com/task/lecture9/form/UpdateFormTest.java

```diff
@@ -0,0 +1,98 @@
+package com.task.lecture9.form;
+
+import org.junit.jupiter.api.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+
+
+class UpdateFormTest {
+
+    @Test
+    void Vinylデータが全て正しく入力されバリデーションエラーとならないこと() {
+        ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory();
+        Validator validator = validatorFactory.getValidator();
+        UpdateForm form = new UpdateForm("aa", "bb", "cc", "2000");
+
+        Set<ConstraintViolation<UpdateForm>> violations = validator.validate(form);
+        assertThat(violations.size()).isEqualTo(0);
+    }
+    @Test
+    void 各項目にnullを入力した際にバリデーションエラーとならないこと() {
+        ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory();
+        Validator validator = validatorFactory.getValidator();
+        UpdateForm form = new UpdateForm(null, null, null, null);
+
+        Set<ConstraintViolation<UpdateForm>> violations = validator.validate(form);
+        assertThat(violations.size()).isEqualTo(0);
+
+    }
+    @Test
+    void 各項目に空文字を入力した際にreleaseYearのみバリデーションエラーとなること() {
+        ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory();
+        Validator validator = validatorFactory.getValidator();
+        UpdateForm form = new UpdateForm("", "", "", "");
+
+        Set<ConstraintViolation<UpdateForm>> result = validator.validate(form);
+        assertThat(result.size()).isEqualTo(1);
+
+        assertThat(result).extracting(propertyPath -> propertyPath.getPropertyPath().toString(),
+                        message -> message.getMessage())
+                .containsOnly(tuple("releaseYear", "整数4桁で入力してください"));
+    }
+    @Test
+    void 各項目に制限文字数以上で入力した際にバリデーションエラーとなること() {
+        ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory();
+        Validator validator = validatorFactory.getValidator();
+        UpdateForm form = new UpdateForm(
+                "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
```

Comment:
↓とすると可読性が高まります。
```suggestion
                "a".repeat(51),
```
https://docs.oracle.com/javase/jp/17/docs/api/java.base/java/lang/String.html#repeat(int)

Reply:
あと、ちょうど50桁のときにバリデーションエラーとならないことの確認もあると安心です。